udemy_exercises/songs/Album.java

```diff
@@ -1,5 +1,71 @@
-package udemy_exercises.songs;
 
-class Album {
+import java.util.ArrayList;
+import java.util.LinkedList;
+
+public class Album {
+    private String name;
+    private String artist;
+    private ArrayList<Song> songs;
+
+    public Album(String name, String artist) {
+        this.name = name;
+        this.artist = artist;
+        this.songs = new ArrayList<Song>();
+    }
+
+    public boolean addSong(String title, double duration) {
+        if(findSong(title) == null) {
+            this.songs.add(new Song(title, duration));
+            return true;
+        }
+        return false;
```

Comment:
As before, this will currently return 'false' without indication of whether it failed due to the song already existing, or because of a failure attempting to add the new Song to the this.songs.